web-browsers/browsers-and-dev-tools.md

```diff
@@ -0,0 +1,220 @@
+# Browsers and Dev Tools
+
+## Vocabulary and Synonyms
+
+| Vocab     | Definition                              | Synonyms | How to Use in a Sentence                                                                                                                   |
+| --------- | --------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| Rendering | Displaying something visual to the user | Showing  | "How the Firefox browser renders that web page may be different compared to how Google Chrome renders it. That's why they look different." |
+
+## The Browser Platform
+
+The _browser_ is the platform that runs web apps in front of an end user.
+
+When we develop the front-end layer for our web apps in code, the _browser_ is the platform that _renders_ it for the user. **Rendering** is the process of displaying it to the user.
+
+As web developers, familiarity with the browser will help us understand its abilities and limitations. Understanding what the browser can and can't do will help us make usable, exciting, and consistent web apps!
```

Comment:
not sure whether consistent means we can access the site consistently, or whether we are building web sites that resemble one another. Suggest 'reliable' instead.
```suggestion
As web developers, familiarity with the browser will help us understand its abilities and limitations. Understanding what the browser can and can't do will help us make usable, exciting, and reliable web apps!
```